community/index.md

```diff
@@ -16,6 +16,11 @@ waiting to be read.
 For fastest answers, join our [Scala.js Gitter chat room](https://gitter.im/scala-js/scala-js) and ask your questions,
 or just chat with other Scala.js developers! There are currently over 500 users in the chat room.
 
+### [`#scala-js` on Discord](https://discord.com/invite/scala)
+
+The [official Scala discord](https://discord.com/invite/scala) has over 600 members.
```

Comment:
That figure is going to get obsolete before we know it. I suggest not to include that.

Reply:
I think it's here because the gitter channel above also has it :)

Reply:
indeed, I was inspired by the above :)

Reply:
gone now